examples/app.py

```diff
@@ -137,7 +130,7 @@ def index():
     search = search.sort(
         request.values.get('sort', 'title')
     )
-    search = ExampleSearch.faceted_search(
-        search=search
-    )
-    return jsonify(search.execute().to_dict())
+    search = ExampleSearch.faceted_search(search=search)
+    results = search.execute().to_dict()
+    results.pop('_faceted_search', None)
```

Comment:
Why are we doing this? Is it related to the ES2-> es5 transition?

Reply:
This is indeed because of ES5, this field does not exist in ES2.
Let's just add a comment:
```Remove _faceted_search for this specific serialization as jsonify cannot serialize its value.```